lametro/templates/base.html

```diff
@@ -3,6 +3,10 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <!-- Google tag (gtag.js) -->
+    <script async src=https://www.googletagmanager.com/gtag/js?id=G-9E47Y9QH06></script>
+    <script> window.dataLayer = window.dataLayer || []; function gtag(){dataLayer.push(arguments);} gtag('js', new Date()); gtag('config', 'G-9E47Y9QH06'); </script>
```

Comment:
It's an old pattern, but the GA code to replace is here: https://github.com/Metro-Records/la-metro-councilmatic/blob/caca48b44c120795369e8eff206065a0f6a5ff90/lametro/templates/base.html#L238-L241

Reply:
Ohh I see, thanks! I've got that change in, I imagine I should also remove references to `ANALYTICS_TRACKING_CODE`?